frontend/src/components/Main.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export const Main = () => {
+  return (
+    <div>
+      <h1>HELLO FROM THE OTHER SIDE</h1>
+    </div>
+  )
+}
```

Comment:
You could add a button here to get to the signin page, otherwise you might not know how to proceed to sign in :)